modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,32 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting {sandboxed-containers-first} resources
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have no running pods that use `kata` as the `runtimeClassName`.
+** You have installed the OpenShift CLI (`oc`).
+** You the command-line JSON processor (`jq`) installed.
+** Verify that you have no running pods that use `kata` as the `runtimeClassName` by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -A -o json | jq -r '.items[] | select(.spec.runtimeClassName | test("kata")).metadata.name'
```

Comment:
@c3d, @cpmeadors, @jensfr - Can this be done using jsonpath instead? We have pretty decent guidelines on not providing lines of code where jq is used because we don't want to force people to install jq unnecessarily if this command can be done without JQ.

Reply:
Eek.  Is there no way to select them by label?

Reply:
@mikemckiernan The point is to select them by `runtimeClassName` to make sure nothing using `kata` is still running. Why do you believe labels would be better here?

Reply:
@c3d - is it possible to do this command without JQ?

Reply:
I can try to convert the command to jsonpath but I need a hint how use it

Reply:
@sagidlow I would say its fine given other product documentation are using `jq`.